core/scss/components/card/examples/card.html

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
can you rename this to default.html? Also, can we add the same examples that you did in Angular (content and content2)?